.github/workflows/swap-ci-workflow.yml

```diff
@@ -0,0 +1,17 @@
+name: Swap functional tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+    - master
+    - main
+    - develop
+  pull_request:
+
+jobs:
+  job_functional_tests:
+    uses: LedgerHQ/app-exchange/.github/workflows/reusable_swap_functional_tests.yml@develop
+    with:
+      branch_for_polkadot: ${{ github.ref }}
```

Comment:
Maybe `repo_for_polkadot` could be added too?

Reply:
https://github.com/LedgerHQ/app-tezos-new/blob/develop/.github/workflows/swap.yml#L17

Reply:
Good catch, thanks